package.json

```diff
@@ -14,16 +14,26 @@
   },
   "license": "MIT",
   "dependencies": {
-    "hoek": "2.x.x"
+    "autoprefixer-stylus": "0.8.0",
+    "hoek": "2.x.x",
+    "jade": "1.11.x",
+    "jeet": "6.1.x",
+    "rupture": "0.6.1",
+    "stylus": "0.52.4",
+    "vision": "3.x.x"
   },
   "devDependencies": {
     "code": "1.x.x",
     "hapi": "9.x.x",
+    "inert": "3.x.x",
```

Comment:
Vision and inert I thought should just be dev dependencies. I think the server.dependency thing handles them being loaded before our plugin code runs. hapi-swagger has inert and vision as dev dependencies only and I'm modeling our approach on that.